src/decimal.erl

```diff
@@ -243,13 +256,13 @@ zero_exp_(Base, Exp) -> {Base, Exp}.
 %% =============================================================================
 
 -spec pow_of_ten(non_neg_integer()) -> pos_integer().
-pow_of_ten(N) ->
-    int_pow(10, N).
 
-int_pow(X, 0) when is_integer(X) ->
-    1;
-int_pow(X, N) when is_integer(X), is_integer(N), N > 0 ->
-    int_pow(X, N, 1).
+pow_of_ten(101) -> %speedup for default precision settings
+    100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
```

Comment:
We do override default precision on each project level, so maybe it's better to use something like 
https://github.com/uwiger/parse_trans/blob/master/doc/ct_expand.md for compile time optimisation